### ISSUE TITLE
Changes in the date of receipt 

### ISSUE THREAD

**Issue by Fashak43** [gibbed/SteamAchievementManager#442]
Is there any way to add a modified achievement date (I use the services of an interpreter because I don't speak English Well)

**Comment by gibbed** [comment]
No. The Steamworks API that SAM uses does not offer this functionality.

**Comment by Fashak43** [comment]
> Нет. API Steamworks, который использует SAM, не предлагает такую ​​функциональность.

Will it ever be possible to do this? Or is it impossible?
